src/main/default/classes/RatingsHandler.cls

```diff
@@ -0,0 +1,10 @@
+public with sharing class RatingsHandler {
+    @AuraEnabled(cacheable=false)
+    public static List<Rating__c> getRating(String documentId) {
```

Comment:
Actually the way am implementing it is that every document suggestion has one custom rating object linked to it with the document id. and in this object we have the score field which represents the amount of people who found that document suggestion helpful. so  this getRating method is not returning all the ratings of a document suggestion cause there is just one and only  rating object corresponding to a document suggestion and in this object we found the score we need. I'll fix the signature of this method to make it clearer, it should return an object instead of a list containing only one object.
and this object is something like that:
{
  score: 1234,
  documentId: "ID of the document suggestion"
}
the score always being the number of people who upvoted this document.

Reply:
Ah perfect! Thanks